### PR TITLE
refactored logging configuration and added more tests

### PR DESCRIPTION
**Pull request by lyoshenka**
makes setup_logging() params clearer. now you know which parts of `args` and `conf` are actually used and what they do

### CODE REVIEW

**Comment by lyoshenka** [comment]
i think the travis error is not real

File: lbry/lbry/extras/cli.py [hunk 1]
```diff
@@ -143,7 +144,7 @@ def _granular_action_groups(self):
 
     def error(self, message):
         self.print_help(argparse._sys.stderr)
-        self.exit(2, '\n'+message+'\n')
+        self.exit(2, '\n' + message + '\n')
```
Comment (jackrobison) [inline lbry/lbry/extras/cli.py]: this can be an f-string

File: lbry/lbry/extras/cli.py [hunk 2]
```diff
@@ -221,43 +222,44 @@ def ensure_directory_exists(path: str):
...
     logging.getLogger('aioupnp').setLevel(logging.WARNING)
     logging.getLogger('aiohttp').setLevel(logging.CRITICAL)
 
     log.setLevel(logging.INFO)
-    if args.verbose is not None:
+    if verbose is not None:
         loop.set_debug(True)
-        if len(args.verbose) > 0:
-            for module in args.verbose:
+        if len(verbose) > 0:
```
Comment (jackrobison) [inline lbry/lbry/extras/cli.py]: this if isn't needed

**Comment by eukreign** [comment]
i'm not ignoring this PR, it's just conflicting with a higher priority one (one that's breaking travis: #2559 ) in the same area of code, i'd like to get that one merged first before looking at this one.

**Comment by lyoshenka** [comment]
ok. i addressed jack's comments. merge when ready

**Comment by eukreign** [comment]
@lyoshenka need rebasing

**Pull request merged**